src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/troubleshooting/missing-entities-relationships.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Troubleshoot OpenTelemetry: Missing entities or relationships"
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Troubleshooting
+metaDescription: Troubleshoot tips for when you use OpenTelemetry with New Relic
+freshnessValidatedDate: 2024-05-01
+---
+
+## Problem
+
+You sent OpenTelemetry data from a service or infrastructure component, and either the [entity](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/) or its [relationships](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/#related-entities) are missing or incorrect in the UI.
+
+## Solution
+
+OpenTelemetry entities are synthesized based on the public rules described for the [`EXT-SERVICE`](https://github.com/newrelic/entity-definitions/blob/b8e75d839eed7859044a9bb601a62e5323107350/definitions/ext-service/definition.yml) entity type. The standard rule to match relies on the presence of the `service.name` dimension (which follows the OpenTelemetry [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/527206ea49e384de957eda105d9425aa8fefca25/specification/overview.md#semantic-conventions)).
```

Comment:
"The standard rule to match relies on the presence of the `service.name` dimension (which follows the OpenTelemetry [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/527206ea49e384de957eda105d9425aa8fefca25/specification/overview.md#semantic-conventions))" is a bit too passive. How about this revision for clarity?

```suggestion
OpenTelemetry entities are synthesized based on the public rules described for the [`EXT-SERVICE`](https://github.com/newrelic/entity-definitions/blob/b8e75d839eed7859044a9bb601a62e5323107350/definitions/ext-service/definition.yml) entity type. How the entity name appears in New Relic is based on the `service.name` dimension, which follows the OpenTelemetry [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/527206ea49e384de957eda105d9425aa8fefca25/specification/overview.md#semantic-conventions)).
```